input/templates/my-account-new-address.hbs

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {{> common/htmlhead}}
+</head>
+
+<body>
+  <div class="darkbg hidden"></div>
+  {{> common/header}}
+  <div class="address-book">
+    <div class="container">
+      <div class="my-account-title">
+        <span class="my-account-title-text icon-user">{{i18n "my-account:myAccount.title"}}</span>
+      </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="address-book-wrapper">
+            <div class="row">
+              <div class="col-sm-3">
+                {{> myaccount/my-account-sidebar positionTwo=true}}
+              </div>
+              <div id="my-account-desktop-content" class="col-sm-9">
+                <div class="address-book-content edit-address-book-content">
+                  <form id="form-add-my-address-book" action="{{@root.meta._links.myAddressBookAddAddressSubmit.href}}" method="POST">
+                    <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
+                    <div class="row">
+                      <div class="col-sm-12">
+                        <div class="address-book-title hidden-xs">
+                          {{i18n "my-account:addressBook.title"}}
+                        </div>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-sm-12">
+                        {{> myaccount/my-address-book-address-form form=content.addressForm}}
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-sm-12">
+                        <div class="edit-address-book-footer">
+                          <div class="btn-grey push-left">
+                            <a href="{{@root.meta._links.myAddressBook.href}}">{{i18n "my-account:buttons.cancelAddressBtn"}}</button>
+                          </div>
+                          <div class="add-new-address-btn push-right">
+                            <button type="submit">{{i18n "my-account:buttons.saveAddressBtn"}}</button>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </form>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  {{!-- {{> common/wishlist}} --}}
+  {{> common/footer}}
+  {{> common/htmlscripts}}
+</body>
+
+</html>
```

Comment:
There was missing a page to create a new address. As it can be very similar to the "edit address" but not exactly the same (the form will submit to another URL and this address doesn't have an ID because it does not exist yet) then I have created a similar page reusing the partial template for the form.